### PR TITLE
Step3

### PR DESCRIPTION
**Pull request by owy1**
built models, added detail and update views

### CODE REVIEW

File: learning_journal/development.ini [hunk 1]
```diff
@@ -14,7 +14,8 @@ pyramid.default_locale_name = en
 pyramid.includes =
     pyramid_debugtoolbar
 
-sqlalchemy.url = sqlite:///%(here)s/learning_journal.sqlite
+#sqlalchemy.url = postgres://kurtrm:hofbrau@localhost:5432/learning_journal
+sqlalchemy.url = postgres:///entries
```
Comment (pasaunders) [inline learning_journal/development.ini:18]: Database won't initialize. It throws an attribute error.

File: learning_journal/learning_journal/views/default.py [hunk 2]
```diff
@@ -14,11 +14,8 @@
...
-    return {
-        'title': 'Main',
-        'entries': ENTRIES
-    }
-
+    entries = request.dbsession.query(Entry).all()
+    return {'entries': entries}
 
 @view_config(
     route_name='detail_view',
```
Comment (pasaunders) [inline learning_journal/learning_journal/views/default.py:21]: Expand your views to meet spec. We need an edit view and a notfound view. You should also remove the corpse code in your default.py.

**Review by pasaunders — COMMENTED** [review]
Get your tests passing and expand their scope. I want to see edge cases and expected failures accounted for.
